EN/02-docker/what-is-docker-.adoc

```diff
@@ -0,0 +1,18 @@
+Introducing Docker and Containers
```

Comment:
I copied a version in  README but keps the one and rename the section `Introduction` . We will decide which is kept finally